src/systemd.py

```diff
@@ -0,0 +1,13 @@
+# Equivalent to the defined variables in systemd.c

```

Comment:
Let's ignore the python stuff until we get the C code nailed down better.

Reply:
Agreed